src/Sarif.Driver.UnitTests/Sdk/FormatForVisualStudioTests.cs

```diff
@@ -69,96 +69,96 @@ public class FormatForVisualStudioTests
             // Test each ResultKind value.
             new object[]
             {
-                ResultKind.Error,
+                ResultLevel.Error,
                 MultiLineTestRegion,
                 $"{TestAnalysisTarget}(2,4,3,5): error {TestRuleId}: First: 42, Second: 54"
             },
 
             new object[]
             {
-                ResultKind.ConfigurationError,
+                ResultLevel.Error,
                 MultiLineTestRegion,
                 $"{TestAnalysisTarget}(2,4,3,5): error {TestRuleId}: First: 42, Second: 54"
             },
 
             new object[]
             {
-                ResultKind.InternalError,
+                ResultLevel.Error,
                 MultiLineTestRegion,
                 $"{TestAnalysisTarget}(2,4,3,5): error {TestRuleId}: First: 42, Second: 54"
             },
 
```

Comment:
Since there isn't a "ResultLevel.InternalError" or ConfigurationError, maybe the right thing to do is just delete these two tests.